apps/gitness/src/framework/routing/utils.ts

```diff
@@ -77,3 +77,20 @@ export const getRouteMapping = ({
   const routeEntries = generateRouteEntries({ routes, parentPath, parentName })
   return generateRouteNameToPathFunctions(routeEntries)
 }
+
+export const extractRedirectRouteObjects = (routes: CustomRouteObject[]): CustomRouteObject[] => {
```

Comment:
this is ripe for adding some unit tests 😅

Reply:
Definitely